lib/check-govuk-and-hmrc-frontend-versions.js

```diff
@@ -0,0 +1,39 @@
+const fs = require('fs');
+const path = require('path');
+
+const projectDir = process.env.INIT_CWD;
+const hmrcFrontendDir = path.join(projectDir, 'node_modules/hmrc-frontend/');
+let govukFrontendVersion = '';
+
+try {
+  govukFrontendVersion = JSON.parse(fs.readFileSync(path.join(hmrcFrontendDir, 'package.json'), 'utf8')).dependencies['govuk-frontend'].replace(/[^0-9.]/g, '');
```

Comment:
I know we were doing it in the old script, but I'm not sure why, I think we could use require() to import this

readFileSync won't cache the result, but since this script is just getting run in a subprocess or tests that doesn't seem like it's much of an issue

that said, probably not worth refactoring